packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx

```diff
@@ -83,7 +82,7 @@ it('renders the banner properly for a deceased person with additional applicatio
       gender="Male"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
       personName="Johnathon Doe"
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```

Comment:
Should this include the img src?
e.g. `<img alt="My Cat" src={demographicsCat} />`